ApiCheck.Test/Builder/ApiBuilder.cs

```diff
@@ -56,6 +56,20 @@ public ApiBuilder Enum(string name = "MyEnum", Tuple<string, int>[] values = nul
       return this;
     }
 
+    public ApiBuilder Enum(string name = "MyEnum", Tuple<string, byte>[] values = null)
```

Comment:
Prefer to make this a generic overload that is called by the default int version:

```
    public ApiBuilder Enum(string name = "MyEnum", Tuple<string, int>[] values = null)
    {
      return Enum<int>(name, values);
    }

    public ApiBuilder Enum<T>(string name = "MyEnum", Tuple<string, T>[] values = null)
    {
      var enumBuilder = _moduleBuilder.DefineEnum(name, TypeAttributes.Public, typeof(T));
      if (values != null)
      {
        foreach (var val in values)
        {
          enumBuilder.DefineLiteral(val.Item1, val.Item2);
        }
      }
      enumBuilder.CreateType();
      return this;
    }```